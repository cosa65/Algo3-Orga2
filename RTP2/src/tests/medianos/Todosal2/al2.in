1 -1 1 al2.txt 0.00001
