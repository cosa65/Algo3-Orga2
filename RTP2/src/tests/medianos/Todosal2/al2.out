0
1
0
0
0
0
0.447214
0
0.447214
0.447214
0.447214
0.447214
0.117647
0.411764
0.117647
0.117647
0.117647
0.117647
0.105264
0.473682
0.105264
0.105264
0.105264
0.105264
0.0975615
0.512192
0.0975615
0.0975615
0.0975615
0.0975615
0.0930225
0.534888
0.0930225
0.0930225
0.0930225
0.0930225
